third_party/2/concurrent/futures/__init__.pyi

```diff
@@ -1,5 +1,9 @@
+import types
 from typing import TypeVar, Generic, Any, Iterable, Iterator, Callable, Tuple, Union
 
+// Old-style exceptions (which were removed in Python 3.x) are instances of
+// types.InstanceType.  See https://github.com/agronholm/pythonfutures/pull/58
+_ExceptionType = Union[BaseException, types.InstanceType]
```

Comment:
This basically means "any old-style class or BaseException" so it provides a very broad spectrum. I'm tempted to just advise putting "Any" instead and be done with it.